packages/litexa-html/README.md

```diff
@@ -1,24 +1,20 @@
-# Litexa HTML Support
-
-This module adds support for the Alexa Web API for Games.
+# Litexa HTML Web API Support
 
 The Alexa Web API for Games allows you to use existing web technologies and tools to create
-visually rich and interactive voice-controlled game experiences. You'll be able to build
+visually rich and interactive voice-controlled game experiences. You can build
 for multimodal devices using HTML and web technologies like Canvas 2D, WebAudio, WebGL,
 JavaScript, and CSS, starting with Echo Show devices and Fire TVs.
 
-**NOTE**: The Alexa Web API for Games is currently in Developer Preview. To gain access
-to its features & documentation, visit the [Alexa Developer
-Blog](https://developer.amazon.com/en-US/blogs/alexa/alexa-skills-kit/2019/11/apply-for-the-alexa-web-api-for-games-developer-preview)
-for sign-up information.
+For more information, see the [Alexa developer portal documentation](https://developer.amazon.com/en-US/docs/alexa/web-api-for-games/understand-alexa-web-api-for-games.html)
 
-## Installation
+There are two broad activities to creating a Web API capable skill: starting a web page and 
+communicating with it. In this extension, new statements and an `HTML` API object 
```

Comment:
Maybe call out that the 2nd one is optional? Don't necessarily need to communicate with the runtime after launching it.

Reply:
Eh, I think I'd call it out of it were mandatory, but it should be reasonably self evident as you're experimenting that the second isn't actually necessary if all you're doing is displaying a static page.